release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -206,6 +207,10 @@ With this update, the External DNS Operator provides the following enhancements:
 Ensure that you migrate from TP to GA. The upstream version of `ExternalDNS` for an {product-title} 4.11 is `v0.12.0` and for the TP is `v0.10.2`.
 For more information, see xref:../networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.adoc#nw-installing-external-dns-operator-on-cloud-providers[About the External DNS Operator].
 
+[id="ocp-4-11-cidr-ranges-for-network"]
+==== {product-title} CIDR Ranges for Networks
+Be aware that CIDR ranges for networks are not adjustable after cluster installation. Red Hat does not provide direct guidance on determining the size as it requires careful consideration of the number of pods created.
```

Comment:
```suggestion
Be aware that CIDR ranges for networks are not adjustable after cluster installation. Red Hat does not provide direct guidance on determining the size because it requires careful consideration of the number of pods created.
```